assets/tests/components/mapPage.test.tsx

```diff
@@ -1587,7 +1587,7 @@ describe("<MapPage />", () => {
           })
           expect(selectedVehicleLabel).toBeInTheDocument()
           // this should be expressed via some accessibility property
-          expect(selectedVehicleLabel).toHaveClass("selected")
+          expect(selectedVehicleLabel.querySelector(".selected")).toBeVisible()
```

Comment:
Related to my [previous comment](https://github.com/mbta/skate/pull/2860/files#r1808661770), it does seem that the vehicle label has remained the same, but not the selected vehicle itself

Reply:
I'm not sure I understand the phrasing here?

Reply:
I don't think there was a regression we could've caught, because it was all CSS.